lib/Controller/RoomController.php

```diff
@@ -1698,4 +1700,16 @@ public function resendInvitations(?int $attendeeId): DataResponse {
 		}
 		return new DataResponse();
 	}
+
+	/**
+	 * @PublicPage
+	 * @RequireModeratorParticipant
+	 */
+	public function setTimeToLive(int $ttl): DataResponse {
+		if ($ttl < 0) {
+			return new DataResponse([], Http::STATUS_BAD_REQUEST);
+		}
+		$this->roomService->setTimeToLive($this->room, $this->userId, $ttl);
```

Comment:
```suggestion
		$this->roomService->setTimeToLive($this->room, $this->participant, $ttl);
```

Otherwise it can not be used by guest moderators